src/config_file.rs

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of Edgehog.
+ *
+ * Copyright 2022 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use edgehog_device_runtime::{error::DeviceManagerError, DeviceManagerOptions};
+use log::info;
+
+pub fn config_file_to_options(
+    custom_config_file_path: Option<String>,
+) -> Result<DeviceManagerOptions, DeviceManagerError> {
+    let paths = ["device-manager.conf", "/etc/device-manager.conf"]
```

Comment:
I would call this `/etc/edgehog.conf` instead.

Reply:
what `custom_config_file_path` is meant to be?

Reply:
> what `custom_config_file_path` is meant to be?

pass a custom path from the command line

Reply:
most programs have different paths for the config file, and a manual override
the first has priority

Reply:
I think we should use two paths, a default and a custom path. If set the custom wins on the default. What do you think @bettio ?